src/Symfony/Component/Routing/Loader/AnnotationDirectoryLoader.php

```diff
@@ -38,7 +38,15 @@ public function load($path, $type = null)
 
         $collection = new RouteCollection();
         $collection->addResource(new DirectoryResource($dir, '/\.php$/'));
-        $files = iterator_to_array(new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($dir), \RecursiveIteratorIterator::LEAVES_ONLY));
+        $files = iterator_to_array(new \RecursiveIteratorIterator(
+            new \RecursiveCallbackFilterIterator(
+                new \RecursiveDirectoryIterator($dir),
+                function (\SplFileInfo $current) {
+                    return '.' !== $current->getBasename()[0];
+                }
+            ),
+            \RecursiveIteratorIterator::LEAVES_ONLY
```

Comment:
you can still add `FilesystemIterator::SKIP_DOTS`
what about `FilesystemIterator::FOLLOW_SYMLINKS` also?